homeassistant/components/zeroconf/__init__.py

```diff
@@ -45,7 +45,11 @@
 ATTR_PROPERTIES = "properties"
 
 ZEROCONF_TYPE = "_home-assistant._tcp.local."
-HOMEKIT_TYPE = "_hap._tcp.local."
+HOMEKIT_TYPES = [
+    "_hap._tcp.local.",
+    # Thread based devices
+    "_hap._udp.local.",
```

Comment:
I've always thought this was weird that `_udp` means everything but `_tcp` here, but I guess that is just evolution of the spec.